Documentation/UserTsconfig/TcaDefaults.rst

```diff
@@ -60,3 +60,7 @@ Example:
         of an extension::
 
             $GLOBALS['TCA']['pages']['ctrl']['useColumnsForDefaultValues'] = 'doktype,fe_group';
+            
+         To avoid that other values are removed from the record you can use this code:
+         
+            $GLOBALS['TCA']['pages']['ctrl']['useColumnsForDefaultValues'] = str_replace(',hidden',',',$GLOBALS['TCA']['pages']['ctrl']['useColumnsForDefaultValues']);
```

Comment:
Would it also be ok to use this example instead?

`$GLOBALS['TCA']['pages']['ctrl']['useColumnsForDefaultValues'] = 'hidden,' . $GLOBALS['TCA']['pages']['ctrl']['useColumnsForDefaultValues'];`

I would guess that's easier to understand.

You approach would replace all `,` with the new field.

Reply:
My approach replace `hidden,` with `,`.
The target is to remove the `hidden` field. Your approach would add `hidden,`which is already in it by default.

Reply:
The code will only work if `hidden` is preceded by a comma. 

Is there a better way to do this?

Reply:
You're total right. Maybe this could be a legit approach.
```
$useColumnsForDefaultValues = $GLOBALS['TCA']['pages']['ctrl']['useColumnsForDefaultValues'];
$newValues = implode(array_diff(str_getcsv($useColumnsForDefaultValues), ["hidden"]),",");
$GLOBALS['TCA']['pages']['ctrl']['useColumnsForDefaultValues'] = $useColumnsForDefaultValues;
```